### PR TITLE
Import color wheels from empyre instead of using a copy

### PR DESCRIPTION
**Pull request by uellue**
Empyre has been released since the copy was created and we can use it
directly now.

LiberTEM contained a custom 2D version. The downstream code is now adapted to use the
3D visualization functions of empyre by setting z to 0.

Since empyre creates an ndarray from the input tuple, having unequal length
or incompatible shapes will trigger "Creating an ndarray from ragged nested
sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with
different lengths or shapes) is deprecated." since NumPy 1.19.

For that reason we use `np.broadcast_arrays()` to make the shapes and sizes match.

Refs #801 

Not sure if it needs a changelog entry since it is an internal API?

CC @jan-car :-)

## Contributor Checklist:

* [x] I have added or updated my entry in [the creators.json file](https://github.com/LiberTEM/LiberTEM/blob/master/packaging/creators.json)
* [x] I have added [a changelog entry](https://github.com/LiberTEM/LiberTEM/tree/master/docs/source/changelog) for my contribution
* [ ] I have added/updated documentation for all user-facing changes
* [x] I have added/updated test cases

### CODE REVIEW

**Comment by sk1p** [comment]
> LiberTEM contained a custom 2D version. The downstream code is now adapted to use the
3D visualization functions of empyre by setting z to 0.

Ah, right, we did change something! Looks good!

> Since empyre creates an ndarray from the input tuple, having unequal length
or incompatible shapes will trigger "Creating an ndarray from ragged nested
sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with
different lengths or shapes) is deprecated." since NumPy 1.19.

Maybe also could be fixed in empyre? It could do the broadcasting internally...

> Not sure if it needs a changelog entry since it is an internal API?

Don't think it needs one, true.

**Comment by uellue** [comment]
> Maybe also could be fixed in empyre? It could do the broadcasting internally...

https://iffgit.fz-juelich.de/empyre/empyre/-/merge_requests/33

**Comment by uellue** [comment]
It looks like `empyre` won't install on Python 3.6?

**Comment by uellue** [comment]
> It looks like `empyre` won't install on Python 3.6?

https://iffgit.fz-juelich.de/empyre/empyre/-/blob/master/setup.cfg#L36

...what to do about that?

**Comment by twentyse7en** [comment]
- [ ] Fix the tests, tests/template/test_radial_fourier_template.py, and tests/template/test_com_template.py, which requires `empyre` as a dependency.
- [ ] Remove the note mentioning Python 3.7 and `empyre` as a requirement for visualization from radialfourier.py and com.py.

**Comment by uellue** [comment]
Bumped to 0.7milestone since @jan-car is on vacation, we likely need time on empyre side, and we don't have time pressure here since the current state works perfectly well. :-)

**Comment by sk1p** [comment]
What's the status on this one - should we do an empyre release on Thursday?

**Comment by jan-car** [comment]
Very nice to see this come to fruition 😄 🎉 !

File: tests/template/test_com_template.py [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-import pytest
+
 import numpy as np
 import nbformat
 from temp_utils import _get_hdf5_params
 from libertem.web.notebook_generator.notebook_generator import notebook_generator
 from nbconvert.preprocessors import ExecutePreprocessor
 
 
-@pytest.mark.skip(reason="requires `empyre`")
```
Comment (sk1p) [inline tests/template/test_com_template.py:11]: The same needs to be done in `test_radial_fourier_template`

**Review by sk1p — CHANGES_REQUESTED** [review]

File: tests/template/test_com_template.py (hunk 1 above)
Comment (uellue) [reply tests/template/test_com_template.py:11]: Thx! I'll look into that later. Just saw the note from @twentyse7en above, will follow up on that.

**Comment by uellue** [comment]
Actually, it should be (data.real, data.imag, 0) since empyre is x, y, z and the imaginary part is commonly plotted on the y axis.

**Review by sk1p — APPROVED** [review]

**Comment by sk1p** [comment]
Merged, thank you!

**Pull request merged**